webapp/src/templates/partials/analytics/target_aggregates.html

```diff
@@ -0,0 +1,38 @@
+<div class="page" ng-show="analyticsTargetAggregatesCtrl.aggregatesDisabled || analyticsTargetAggregatesCtrl.loading">
+  <div class="item-content empty-selection disabled" ng-show="analyticsTargetAggregatesCtrl.aggregatesDisabled">
+    <div translate>analytics.target.aggregates.disabled</div>
+  </div>
+  <div class="item-content empty-selection" ng-show="!analyticsTargetAggregatesCtrl.aggregatesDisabled">
+    <div>
+      <div class="loader"></div>
+    </div>
+  </div>
+</div>
+<div class="page" ng-if="!analyticsTargetAggregatesCtrl.aggregatesDisabled && !analyticsTargetAggregatesCtrl.loading">
+  <div id="target-aggregates-list" class="col-sm-4 inbox-items left-pane">
+    <ul ng-show="!analyticsTargetAggregatesCtrl.loading">
+      <li ng-repeat="aggregate in analyticsTargetAggregatesCtrl.aggregates" data-record-id="{{aggregate.id}}" class="content-row" ng-class="{ 'selected': analyticsTargetAggregatesCtrl.selected && analyticsTargetAggregatesCtrl.selected.id === aggregate.id }">
+        <a href="#analytics/target-aggregates/{{aggregate.id}}">
+          <div class="icon" ng-bind-html="aggregate.icon | resourceIcon"></div>
+          <div class="content">
+            <div class="heading">
+              <h4 ng-if="aggregate.translation_key" translate>{{aggregate.translation_key}}</h4>
+              <h4 ng-if="!aggregate.translation_key">{{aggregate.title | translateFrom}}</h4>
+              <div class="status" ng-class="{'error': aggregate.hasGoal && !aggregate.aggregateValue.goalMet, 'success':  aggregate.hasGoal && aggregate.aggregateValue.goalMet}">
+                <span ng-if="aggregate.hasGoal" translate translate-values="aggregate.aggregateValue">analytics.target.aggregates.ratio</span>
+                <span ng-if="!aggregate.hasGoal && aggregate.isPercent">{{aggregate.aggregateValue.percent}}%</span>
+                <span ng-if="!aggregate.hasGoal && !aggregate.isPercent">{{aggregate.aggregateValue.pass}}</span>
+              </div>
```

Comment:
I think this would be clearer split into two divs, one with `ng-if="aggregate.hasGoal"` and with `ng-if="!aggregate.hasGoal"` - there's virtually no code shared between the two paths.